intel_owl/settings.py

```diff
@@ -92,6 +92,7 @@ def retrieve(self, file, analyzer):
     "django.contrib.contenttypes",
     "django.contrib.sessions",
     "django.contrib.messages",
+    "django.contrib.staticfiles",
```

Comment:
@Eshaan7 is this really necessary? I think Simone deleted it because those static files are just used in the readme

Reply:
The django admin breaks without this

Reply:
Because it needs css/JS static files to work

Reply:
ah yeah right thanks to the explanation